pkg/svcat/service-catalog/binding.go

```diff
@@ -264,3 +265,69 @@ func (sdk *SDK) bindingHasStatus(binding *v1beta1.ServiceBinding, status v1beta1
 
 	return false
 }
+
+// RemoveFinalizersForBinding removes all finalizers for a single binding
+func (sdk *SDK) RemoveFinalizersForBinding(namespacedName types.NamespacedName) error {
+	// Get binding object from namespacedName
+	binding, err := sdk.RetrieveBinding(namespacedName.Namespace, namespacedName.Name)
+	if err != nil {
+		return err
+	}
+	finalizers := sets.NewString(binding.Finalizers...)
+	finalizers.Delete(v1beta1.FinalizerServiceCatalog)
+	binding.Finalizers = finalizers.List()
+	_, err = sdk.ServiceCatalog().ServiceBindings(binding.Namespace).UpdateStatus(binding)
+	return err
+}
+
+// RemoveFinalizersForBindings removes all finalizers for the provided list of bindings
+func (sdk *SDK) RemoveFinalizersForBindings(bindings []types.NamespacedName) ([]types.NamespacedName, error) {
+	var g sync.WaitGroup
+	deletedBindings := make(chan types.NamespacedName, len(bindings))
+	errs := make(chan error, len(bindings))
+	for _, binding := range bindings {
+		g.Add(1)
+		go func(binding types.NamespacedName) {
+			defer g.Done()
+			err := sdk.RemoveFinalizersForBinding(binding)
+			if err == nil {
+				deletedBindings <- binding
+			}
+			errs <- err
+		}(binding)
+	}
+
+	g.Wait()
+	close(errs)
+	close(deletedBindings)
+
+	// Collect any errors that occurred into a single formatted error
+	bindErr := &multierror.Error{
+		ErrorFormat: func(errors []error) string {
+			return joinErrors("error:", errors, "\n  ")
+		},
+	}
+	for err := range errs {
+		bindErr = multierror.Append(bindErr, err)
+	}
+
+	//Range over the deleted bindings to build a slice to return
+	deleted := []types.NamespacedName(nil)
+	for b := range deletedBindings {
+		deleted = append(deleted, b)
+	}
+	return deleted, bindErr.ErrorOrNil()
+}
+
+// RemoveFinalizersByInstance removes all finalizers for an instance.
```

Comment:
Remove v1beta1.FinalizerServiceCatalog  from **all bindings** for the specified instance